landing-page/content/common/how-to-release.md

```diff
@@ -173,8 +175,8 @@ Next, you need to close the staging repository:
 
 1. Go to [Nexus](https://repository.apache.org/) and log in
 2. In the menu on the left, choose "Staging Repositories"
-3. Select the Iceberg repository
-   * If multiple staging repositories are created after running the script, set `org.gradle.parallel=false` in `gradle.properties`
+3. Select the Iceberg repository.
```

Comment:
```suggestion
3. Select the Iceberg repository
```

Reply:
I think this was from before?  but fixed.